conference.js

```diff
@@ -930,7 +930,8 @@ export default {
 
                 APP.UI.addLocalStream(stream);
 
-                stream.videoType === 'camera' && APP.UI.enableCameraButton();
+                stream.videoType === 'camera'
+                && APP.UI.disableCameraButton(false);
```

Comment:
Is the above jitsi-meet's way of formatting/indenting `&&` on multiple lines? Or it's missing an indentation level?